packages/host/src/patch.ts

```diff
@@ -417,7 +417,7 @@ export async function patch(savePath: string, modules: SaveModule[]) {
 	root.file("clusterio.json", JSON.stringify(patchInfo, null, "\t"));
 
 	// Write back the save
-	let tempSavePath = `${savePath}.tmp`;
+	let tempSavePath = savePath.replace(/(\.zip)?$/, ".tmp.zip");
```

Comment:
Is there a reason you have used `.replace` here when in the other two cases you have used `path.parse`?

Reply:
It matches the way it's done for Factorio save paths in other places like Hosts.ts https://github.com/clusterio/clusterio/blob/faadf873fb98e070c0732634de4266d5ff90c055/packages/host/src/Host.ts#L754